libraries/management/commands/import_library_versions.py

```diff
@@ -0,0 +1,137 @@
+import djclick as click
+
+from libraries.github import GithubAPIClient, GithubDataParser, LibraryUpdater
+from libraries.models import Library, LibraryVersion
+from versions.models import Version
+
+
+@click.command()
+@click.option("--token", is_flag=False, help="Github API token")
+@click.option("--release", is_flag=False, help="Boost version number (example: 1.81.0)")
+def command(release, token):
+    """Cycles through all Versions in the database, and for each version gets the
+    corresponding tag's .gitmodules.
+
+    The command then goes to the same tag of the repo for each library in the
+    .gitmodules file and uses the information to create LibraryVersion instances, and
+    add maintainers to LibraryVersions.
+
+    Args:
+        token (str): Github API token, if you need to use something other than the setting.
+        release (str): Boost version number (example: 1.81.0). If a partial version number is
+        provided, the command process all versions that contain the partial version number
+        (example: "--version="1.7" would process 1.7.0, 1.7.1, 1.7.2, etc.)
+    """
+    client = GithubAPIClient(token=token)
+    parser = GithubDataParser()
+    updater = LibraryUpdater(client=client)
+
+    skipped = []
+
+    if release is not None:
+        versions = Version.objects.filter(name__icontains=release)
+    else:
+        versions = Version.objects.active()
+
+    for version in versions:
+        click.echo(f"Processing version {version.name}...")
+
+        # Get the .gitmodules for the version using the version name, which is also the git tag
+        ref = client.get_ref(ref=version.name)
+        try:
+            raw_gitmodules = client.get_gitmodules(ref=ref)
+        except Exception:
+            # Only happens for one version; uncertain why.
+            click.secho(f"Could not get gitmodules for {version.name}.", fg="red")
+            skipped.append(
+                {"version": version.name, "reason": "Could not get gitmodules"}
+            )
+            continue
+
+        gitmodules = parser.parse_gitmodules(raw_gitmodules.decode("utf-8"))
+
+        for gitmodule in gitmodules:
+            click.echo(f"Processing module {gitmodule['module']}...")
+            if gitmodule["module"] in updater.skip_modules:
+                click.echo(f"Skipping module {gitmodule['module']}.")
+                continue
+
+            libraries_json = client.get_libraries_json(repo_slug=gitmodule["module"])
+
+            # If the libraries.json file exists, we can use it to get the library info
+            if libraries_json:
+                libraries = []
+                if type(libraries_json) is list:
+                    for library in libraries_json:
+                        data = parser.parse_libraries_json(library)
+                        libraries.append(data)
+                else:
+                    libraries.append(parser.parse_libraries_json(libraries_json))
```

Comment:
I would suggest a small refactoring of this code, to avoid duplication:

```python
if libraries_json is not iterable:  # whichever check we prefer
    libraries_json = [libraries_json]
libraries = [parser.parse_libraries_json(lib) for lib in libraries_json]
```